FLOWTYPE.md

```diff
@@ -1,21 +1,27 @@
-Using flowtype
-==============
+# Using flowtype
 
 This codebase has been set up to use [flowtype](http://flowtype.org/), a static typechecker for Javascript.
 
-## Setup
+## Installation
 
-To use it, first install `flow`:
+To use it, first install `flow`.
+
+### *nix
 
 ``` sh
 brew update
 brew install flow
```

Comment:
Hehe... I just removed it from `package.json`. :-)

So instead of using `brew`, you think `flow-bin` should just be a part of the standard `package.json` install?

(How will Windows users be affected by this?)